samples/matlab_experimental/test_examples.m

```diff
@@ -1,31 +1,52 @@
-% runs selected examples without pausing
-
-ctLoad
-clear all
-close all
-
-equil();
-isentropic();
-reactor1();
-reactor2();
-surf_reactor;
-periodic_cstr;
-% plug_flow_reactor;
-lithium_ion_battery
-rankine;
-prandtl1();
-prandtl2();
-% flame1;
-flame2;
-catcomb;
-diff_flame;
-ignite;
-ignite_hp;
-ignite_uv;
-diamond_cvd;
-
-clear all
-close all
-ctUnload
-
-disp('Test example run successful.')
+function success = test_examples()
+    ctLoad
+
+    examples = {
+        'equil', 'isentropic', 'reactor1', 'reactor2', 'surf_reactor', ...
+        'periodic_cstr', 'plug_flow_reactor', 'lithium_ion_battery', ...
+        'rankine', 'prandtl1', 'prandtl2', 'catcomb','ignite', 'diamond_cvd', ...
+        % 'flame1', 'flame2', 'diff_flame', 'ignite_hp', 'ignite_uv'
```

Comment:
The last two examples here should be included as they're zero-d reactors, right?